routes/_static/themes.js

```diff
@@ -85,4 +85,15 @@ const themes = [
   }
 ]
 
+if (process.browser && CSS.supports('color', '-moz-field')) {
+  const div = document.createElement('div')
+  div.style.color = '-moz-field'
+
+  themes.push({
+    name: 'native',
+    label: 'Native (beta)',
+    color: window.getComputedStyle(div).color
+  })
+}
```

Comment:
this also needs to go into the `inline-script`. I need to modify the `inline-script` so that it's easier to do this kind of thing.

Reply:
Don't sweat it. I just fixed this in #761